libs/kameleon/project.clj

```diff
@@ -13,7 +13,7 @@
                  [postgresql "9.1-901-1.jdbc4"]
                  [slingshot "0.10.3"]]
   :plugins [[lein-marginalia "0.7.1"]]
-  :manifest {"db-version" "2.1.0:20150810.01"}
+  :manifest {"db-version" "2.1.0:20150810.02"}
```

Comment:
Since this conversion was added today, this version could be bumped to `2.1.0:20150811.01`